File: KrasiMilan/QL/src/main/java/nl/uva/softwcons/ast/expression/binary/arithmetic/MultiplyExpression.java
package nl.uva.softwcons.ast.expression.binary.arithmetic;

import nl.uva.softwcons.ast.expression.Expression;
import nl.uva.softwcons.ast.expression.binary.BinaryExpression;

public class MultiplyExpression extends BinaryExpression {

    public MultiplyExpression(Expression left, Expression right) {
        super(left, right);
    }

}

File: CarlaErik/UvA.SoftCon.Questionnaire/AST/Expressions/Numeric/MultiplyExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UvA.SoftCon.Questionnaire.AST.Expressions.Numeric
{
    public class MultiplyExpression : BinaryNumericExpression
    {
        public MultiplyExpression(INumericExpression left, INumericExpression right)
            : base(left, right) { }

        public override int Evaluate()
        {
            return Left.Evaluate() * Right.Evaluate();
        }
    }
}
